Sources/Foundation/Process.swift

```diff
@@ -499,7 +499,7 @@ open class Process: NSObject {
         }
 
 #if os(Windows)
-        var command: [String] = [launchPath]
+        var command: [String] = [try FileManager.default._fileSystemRepresentation(withPath: launchPath) { String(decodingCString: $0, as: UTF16.self) }]
```

Comment:
Does `_fileSystemRepresentation` need to/can be applied on other platforms as well (just to be consistent)

Reply:
Good point, it doesn't have the same level of effect on other platforms but I'll add it for good measure